GameData/RealismOverhaul/Engine_Configs/AJ10_190_Config.cfg

```diff
@@ -30,9 +30,9 @@
 
     @MODULE[ModuleGimbal],*
     {
-        @gimbalRange = 6.0
-        %useGimbalResponseSpeed = True
-        %gimbalResponseSpeed = 16
+        gimbalRange = 6.0
```

Comment:
Yep, that would be the perfect solution. You could even use a value of 0 for them (since it is only a placeholder).